### ISSUE TITLE
Only show share buttons on candidate that user voted for

### ISSUE THREAD

**Issue by namimody** [DoSomething/voting-app#360]
While I don't think a vote is actually registering (clicking on another cat after submitting one vote is not bringing me to the vote confirmation page) I can click on an unlimited number of cats and get the image below, so I feel like I just voted again and feel like I can keep voting. Perhaps this is just a copy change, idea below:

You already voted today! You can vote again in 24 hours.
Get Miss Antonina Wilderman Sr. votes!

![image](https://cloud.githubusercontent.com/assets/12396076/8257478/d8024544-167b-11e5-96cb-f573df33981c.png)

**Comment by DFurnes** [comment]
Good catch, that language is kinda confusing. That's the messaging shown if a user cannot vote yet, so no votes are being cast when clicking on candidates. If the user is able to vote & logged in, they'll be given a "Vote" button in this container instead.

**Comment by DFurnes** [comment]
@mikefantini Any suggestions on better language to use here?

**Comment by joolaay** [comment]
@DFurnes @namimody the main confusing element is the share - as a user, I shouldn't be given the option of sharing a cat who I didn't vote for. What do you think about removing shares for all cats besides the cat I voted for? 

Theoretically, would users share many cats, or just one cat to which they are loyal?

**Comment by DFurnes** [comment]
Sounds reasonable. Updated the issue title.

**Comment by mikefantini** [comment]
this should hide the shares and the copy "Get [celebrity_name] more votes!" yeah? @DFurnes @julielorch

**Comment by DFurnes** [comment]
Yep, I think that's the plan.
